docs-src/cli/activity/complete.md

```diff
@@ -8,43 +8,30 @@ tags:
 ---
 
 The `temporal activity complete` command completes an [Activity Execution](/concepts/what-is-an-activity-execution).
+Along with completing the Activity, the result given upon return can be set as well.
+
+`temporal activity complete --activity-id=MyActivity --result=ActivityComplete`
```

Comment:
should we put something more random in a result to hint users this is a string/json?